Integrations/RSANetWitness_v11.1/RSANetWitness_v11.1.py

```diff
@@ -108,6 +108,7 @@ def get_token():
 PASSWORD = demisto.params()['credentials']['password']
 USE_SSL = not demisto.params()['insecure']
 VERSION = demisto.params()['version']
+IS_FETCH = demisto.params()['isFetch']
 FETCH_TIME = demisto.params().get('fetchTime', '1 days')
```

Comment:
please change fetchTime to fetch_time according to python pattern, also in yml for the param itself.